### PR TITLE
Clean-up unwanted spaces (Part 1: CI code)

### PR DESCRIPTION
**Pull request by JunielKatarn**
NOTE: This is a formatting-only change.

Remove trailing and unwanted whitespace that could lead to more complex code reviews when files are edited in a non-EditorConfig-enabled IDE.

The recommendation is to edit any non-source files using Visual Studio Code with the official EditorConfig extension enabled:
https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/8403)

### CODE REVIEW

File: .ado/publish.yml [hunk 1]
```diff
@@ -133,8 +133,8 @@ jobs:
 
       - template: templates/yarn-install.yml
         parameters:
-         workingDirectory: $(Build.StagingDirectory)\bot-coordinator
-         frozenLockfile: false
+        workingDirectory: $(Build.StagingDirectory)\bot-coordinator
```
Comment (NickGerleman) [inline .ado/publish.yml:136]: Does this change have a runtime effect?

Comment (JunielKatarn) [reply .ado/publish.yml:136]: It surely will. The formatter believed it was an extra space, where it was a missing one.

Some of these I'll fix manually and carefully.

I wouldn't worry about CI not catching an indentation error. The CI loops just wouldn't make sense if I get those wrong.

**Review by dannyvv — APPROVED** [review]
Thanks for doing this, seeing these PR's always makes me happy :)

**Comment by JunielKatarn** [comment]
> Thanks for doing this, seeing these PR's always makes me happy :)

And it makes me less stressed-out :D

**Pull request merged**